Kconfig.dependencies

```diff
@@ -279,4 +279,15 @@ config SIDEWALK_DFU_SERVICE_BLE_DEFAULTS
 config MCUMGR_TRANSPORT_BT_AUTHEN
 	default n
 
+config SIDEWALK_BLE_RSSI_READ_ON_MESSAGE
+	bool "Read RSSI on received message"
+	help
+	  Add log message with the current average value of RSSI on every received BLE message
+
+config SIDEWALK_BLE_RSSI_AVERAGE_DISABLE
```

Comment:
Isn't It sounds like something configurable on softdevice controller level. Isn't this config redundant to some NCS BLE SDC config?